woocommerce/woocommerce
# Fix js packages lint errors

### All Submissions:






### Changes proposed in this Pull Request:



closed https://github.com/woocommerce/woocommerce/issues/32358

- Update `@woocommerce/eslint-plugin` to use workspace version to run the lint commands.
- Fixes lint errors automatically via `pnpm nx run-many --target=lint --projects=@woocommerce/api,@woocommerce/e2e-environment,@woocommerce/e2e-utils -- --fix` 


### How to test the changes in this Pull Request:

1. Should able to run `pnpm nx run-many --target=lint --projects=@woocommerce/api,@woocommerce/api-core-tests,@woocommerce/e2e-core-tests,@woocommerce/e2e-environment,@woocommerce/e2e-utils` without the errors like:


```
> nx run @woocommerce/api:lint

> @woocommerce/api@0.2.0 lint /Users/chihsuan/Projects/woocommerce/packages/js/api
> eslint src

2. Review the changes and confirm only this PR only reformat the files.
3. See if all actions pass

Oops! Something went wrong! :(

ESLint: 8.2.0

ESLint couldn't find the config "plugin:testing-library/recommended" to extend from. Please check that the name of the config is correct.

The config "plugin:testing-library/recommended" was referenced from the config file in "/Users/chihsuan/Projects/woocommerce/node_modules/.pnpm/@woocommerce+eslint-plugin@1.3.0/node_modules/@woocommerce/eslint-plugin/configs/react-testing-library.js".

If you still have problems, please stop by https://eslint.org/chat/help to chat with the team.

 ELIFECYCLE  Command failed with exit code 2.
```

### Other information:

There are still many errors, but I think we should fix them in other PRs.




### Changelog entry

Dev: reformat js files to fix lint errors

### FOR PR REVIEWER ONLY:
